protocols/grpc/hyperstart.proto

```diff
@@ -42,10 +43,18 @@ message CreateContainerRequest {
 	Spec OCI = 4;
 }
 
-message StartContainerRequest {
+message ContainerRequest {
```

Comment:
no need to rename StartContainerRequest any more.

Reply:
Done.